docs/user-guide/projects/project-settings/files/README.md

```diff
@@ -122,3 +115,12 @@ To delete an existing storage, you can click on the delete icon on the right sid
 To be able to delete a storage from a project you will be asked to confirm the action by introducing the name of the storage. This action is irreversible and will remove all links from work packages of this project to files and folders of that storage. Also, in case this storage has an automatically managed project folder, it and its files will be deleted forever.
 
 ![Delete a storage from a project](storage-delete.png)
+
+## Show attachments in the work packages files tab
```

Comment:
I think this was designed by @psatyal. I didn't find anything strange on it, but maybe he wants to have a look into it.